sound/soc/sof/sof-pci-dev.c

```diff
@@ -158,7 +158,9 @@ int sof_pci_probe(struct pci_dev *pci, const struct pci_device_id *pci_id)
 	sof_pdata->name = pci_name(pci);
 	sof_pdata->desc = desc;
 	sof_pdata->dev = dev;
-	sof_pdata->fw_filename = desc->default_fw_filename;
+
+	sof_pdata->ipc_type = desc->ipc_default;
```

Comment:
Should `sof_pdata->ipc_type = desc->ipc_default;` belong to `ASoC: SOF: add default IPC capability and file paths`?